src/back/routes/proposal.ts

```diff
@@ -564,7 +565,10 @@ export async function updateProposalStatus(req: WithAuth<Request<{ proposal: str
     NotificationService.grantProposalEnacted(proposal)
   }
 
-  ProposalService.commentProposalUpdateInDiscourse(id)
+  const updatedProposal: ProposalAttributes | undefined = await ProposalModel.findOne<ProposalAttributes>({
```

Comment:
Isn't the type being inferred from the `findOne<ProposalAttributes>`? Maybe `findOne<ProposalAttributes | undefined>`?